### PR TITLE
workround for windows 11 bug with RTL in options

### PR DESCRIPTION
**Pull request by adipose**

### CODE REVIEW

File: src/mpc-hc/PPageSheet.cpp [hunk 1]
```diff
@@ -354,6 +354,30 @@ LRESULT CPPageSheet::OnDpiChanged(WPARAM wParam, LPARAM lParam) {
     return FALSE;
 }
 
+int CALLBACK PropSheetCallBackRTL(HWND hWnd, UINT message, LPARAM lParam) {
+    switch (message) {
+        case PSCB_PRECREATE:
+        {
+            //arabic or hebrew
+            if (AfxGetAppSettings().language == 1025 || AfxGetAppSettings().language == 1037) {
```
Comment (clsid2) [inline src/mpc-hc/PPageSheet.cpp]: Maybe add IsLangRTL() helper function?

Comment (adipose) [reply src/mpc-hc/PPageSheet.cpp]: Good idea, but the translations code did it a different way.

```
if (PRIMARYLANGID(languageResource.localeID) == LANG_ARABIC || PRIMARYLANGID(languageResource.localeID) == LANG_HEBREW)
```

I didn't have that option so I did it a different way.

Comment (adipose) [reply src/mpc-hc/PPageSheet.cpp]: One other issue--by the time SetLanguage() is called, appsettings is not updated (see PPagePlayer.cpp lin 186-191).  Need to redo that, too.

**Comment by clsid2** [comment]
Unrelated to this, but I also noticed some minor issues with the playlist in RTL. For example spacing between the numbering and the titles.

**Pull request merged**